mlflow/mleap.py

```diff
@@ -167,3 +169,31 @@ def add_to_model(mlflow_model, path, spark_model, sample_input):
                             mleap_version=mleap.version.__version__,
                             model_data=mleap_datapath_sub,
                             input_schema=mleap_schemapath_sub)
+
+
+def _get_mleap_schema(dataframe):
+    """
+    :param dataframe: A PySpark dataframe object
+
+    :return: The schema of the supplied dataframe, in MLeap format. This serialized object of type
+    `ml.combust.mleap.core.types.StructType`, represented as a JSON dictionary.
+    """
+    from pyspark.ml.util import _jvm
+    ReflectionUtil = _jvm().py4j.reflection.ReflectionUtil
+
+    # Convert the Spark dataframe's schema to an MLeap schema object
```

Comment:
Could you write the equivalent scala code for clarity? I.e., we just want to call

`TypeConverters. sparkSchemaToMleapSchema(dataframe)`

but are impeded by the fact that TypeConverters is an `object` in Scala.